Add pegasus implementations of kBET (#52)
openproblems-bio/task_batch_integration: CONTRIBUTING.md

### Running the benchmark

To locally test the benchmark workflow, you can use the following command:

```bash
scripts/run_benchmark/run_test_local.sh
```

> Other scripts in `scripts/run_benchmark/` provide commands for testing on sequeracloud an for production runs. 

Reviewer: ```suggestion
Other scripts in `scripts/run_benchmark/` provide commands for testing on Seqera Cloud and for production runs. 
```
